packages/blade/src/components/Icons/_Svg/Svg/Svg.native.tsx

```diff
@@ -12,22 +12,25 @@ const StyledSvg = styled(SvgNative)((props) => {
   return styledPropsCSSObject;
 });
 
-const Svg = React.forwardRef<SvgProps, any>(
-  ({ children, height, viewBox, width, fill, ...styledProps }, ref): ReactElement => {
-    return (
-      <StyledSvg
-        {...makeAccessible({ hidden: true })}
-        height={height}
-        viewBox={viewBox}
-        width={width}
-        fill={fill}
-        ref={ref}
-        {...styledProps}
-      >
-        {children}
-      </StyledSvg>
-    );
-  },
-);
+const _Svg: React.ForwardRefRenderFunction<any, SvgProps> = (
```

Comment:
`React.forwardRef<SectionList, ActionListBoxProps>()` this is the direct function invocation. 

This is just the return type of the React.forwardRef 

```
React.ForwardRefRenderFunction<any, SvgProps> 
```

Related to https://github.com/razorpay/blade/pull/1265#discussion_r1227903086

Need to do this way otherwise getting #701